Update links

MattiSG opened the pull request:
As spotted by @LorenzoStacchioDev in https://github.com/openfisca/openfisca-doc/issues/144#issuecomment-392488716, quite a few links were dead.

This changeset updates all links to the OpenFisca websites.

sandcha commented on howto-web-no-local-install.md:
````diff
@@ -21,7 +21,7 @@ Let's see how to use OpenFisca on one of those services: [repl.it](https://repl.
     ```
 
-    > You can also import files (e.g. JSON files describing input [situations](http://openfisca.org/doc/openfisca-web-api/input-output-data.html#describing-the-situation)) by clicking on the `import or drop` button.
+    > You can also import files (e.g. JSON files describing input [situations](./openfisca-web-api/input-output-data.md#describing-the-situation)) by clicking on the `import or drop` button.
````
Will it work with `.md`? Keep a `.html` link?

sandcha replied:
> Add minor fix for `import or drop` and set it to `upload or drop`?

fpagnoux replied:
I think in gitbook all relative links should be in `.md`, and are automatically compiled during the build process 🙂

MattiSG replied:
+1 to what @fpagnoux said 🙂 

Also, if you ever have any doubt about what the output will be, you are very encouraged to compile the book locally with `gitbook serve` 😃

MattiSG replied:
Regarding the suggested “minor fix”, I'd like to keep this PR focused on links update, not on updating a UI I don't know about 🙂

sandcha commented on windows-no-admin.md:
````diff
@@ -72,4 +72,4 @@ To run it, save your modifications, go to the directory containing it, right cli
 ```
 
-In case you run into a problem, you can [open an issue](https://github.com/openfisca/openfisca-france/issues/new) or send an email to `contact@openfisca.fr`.
+In case you run into a problem, please [open an issue](https://github.com/openfisca/openfisca-core/issues/new).
````
Keep email option and set it to `contact@openfisca.org`? 
Why would we close this door to new users for example?

MattiSG replied:
Because:

1. Do you know who reads that email address? 🙂
2. Issues allow for distribution of the support load.
3. How many email contacts have we received through that door in the last three years?

maukoquiroga replied:
I absolutely agree with _1._ in that I'm tired of receiving spam. 😄 

Regarding the change itself, seems to me it is out of scope, and you state it yourself @MattiSG both in the pull request description and in https://github.com/openfisca/openfisca-doc/pull/146/files#r191969020

MattiSG replied:
Yes. I will use less knights when the time to review is less than a week :wink:

sandcha replied:
Ok, the email address wasn't really working here.

MattiSG commented:
Hey @sandcha thanks for the review  🙂 
I'm not clear which one was the change request and which one was the question though 🤔 What's blocking for you?

maukoquiroga commented:
I'm sorry @MattiSG but you **can't** impose a time to review, so the use of your privileges in order to enforce a behaviour that this community hasn't agreed to will invariably be seen as an hostility.

MattiSG commented:
Uh? 😳
I'm sorry, unless I'm mistaken, this PR has been merged 4 days ago, after having been approved by both a member of your team and yourself, who had as much time to review as everyone else in the community.
I addressed feedback.
There has been no discussion on the PR since then.
To the best of my knowledge I have made no use of any admin or social privilege at any point.
Everyone is welcome to check the public history of this PR here and in the Git history.

Could you please help me understand which privileges I have used and what behaviour I have enforced, so that I don't do it again in the future? 🙂 Did you feel forced to approve updating links from HTTP to HTTPS in any way?

————
Written on a phone, please excuse the terseness.
Rédigé sur un téléphone, veuillez excuser la brièveté.



Matti Schneider
mattischneider.fr
@matti_sg / @matti_sg_fr
————
Written on a phone, please excuse the terseness.
Rédigé sur un téléphone, veuillez excuser la brièveté.
> Le 10 juin 2018 à 23:37, Mauko Quiroga <notifications@github.com> a écrit :
> 
> I'm sorry @MattiSG but you can't impose a time to review, so the use of your privileges in order to enforce a behaviour that this community hasn't agreed to will invariably be seen as an hostility.
> 
> —
> You are receiving this because you were mentioned.
> Reply to this email directly, view it on GitHub, or mute the thread.
